backend/Models/Database/ConfirmationStatuses.cs

```diff
@@ -0,0 +1,8 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class ConfirmationStatuses
```

Comment:
The class should be named in the singular, as it represents "a confirmation status"
```suggestion
    public class ConfirmationStatus
```